packages/bygger/src/translations/global/GlobalTranslationsPage.jsx

```diff
@@ -62,6 +62,17 @@ const useGlobalTranslationsPageStyles = makeStyles({
   },
 });
 
+const useSelectedTag = () => {
```

Comment:
Replace all this state with just linking directly to /skjematekster.
If we never link to a page that does not have the tag in the URL, we don't need selectedTag state at all, but can use tag from URL directly.